libs/verify-android-environment.sh

```diff
@@ -22,6 +22,12 @@ if [ -z "${ANDROID_NDK_ROOT}" ]; then
   exit 1
 fi
 
+if [ -z "${ANDROID_NDK_HOME}" ]; then
```

Comment:
Can we also add a check that `$ANDROID_NDK_HOME == $ANDROID_NDK_ROOT`? That could avoid hours of headaches for somebody in the future :)